i18n/fr/docusaurus-plugin-content-docs/version-24.10/releases/centreon-os.mdx

```diff
@@ -16,184 +16,186 @@ Vous trouverez dans ce chapitre tout ce qui concerne **Centreon Open Source**.
-  - [MAP widget] Added an information message displayed when editing the widget.
-  - [Metrics Graph widget] Added a warning message indicating that thresholds have been hidden when selecting two different metric units.
-  - [Metrics Graph widget] Added an information tooltip when hovering over an element on the graph.
-  - [Single Metric widget] Simplified metric selection.
-  - [Single Metric widget] Simplified dataset selection by forcing a "host + service" pair.
-  - [Status Grid widget] Added an "unhandled" state property.
-  - [Status Grid widget] Added a new "condensed" view.
+**Additional Connector Configurations**
 
-> BREAKING CHANGE: The way metrics are handled has been changed to allow for more flexibility (e.g., you now have the possibiliyy to exclude metrics). If you used the metrics graph widget, you may have to select your metrics again.
-
-**User Experience**
-
-- Added full screen mode for the entire application.
+- The **Additional connector configurations** menu allows you to define the credentials to access a monitored resource through the Centreon interface rather than having to connect to your pollers manually.
 
 **API**
 
-- **Dashboards**
-  - Added new endpoints to manage [playlists](centreon-commercial-extensions.mdx#centreon-it-edition-extensions).
-  - Added a new endpoint to duplicate dashboards.
-  - Updated endpoints to handle the Status Grid "condensed" view.
-
-- **Configuration**
-  - Added an endpoint to update a service severity.
-  - Added an endpoint to create Broker outputs.
-  - Added an endpoint to list all installed connectors.
-  - Added an endpoint to list graph templates.
-  - Added an endpoint to list and delete Business Activities.
-  - Added an endpoint to update a service.
-  - Added an endpoint to update a host partially.
-  - Added an endpoint to handle the file format associated with a Stream Connector.
+- Added an endpoint that automatically deploys the services brought by the host template.
+- Added an endpoint to list Pollers.
+- Added an endpoint to create Broker inputs.
+- Added endpoints to create, delete, update and list/get Additional Connector Configurations.
+- Added API endpoints to return the complete hierarchy for a given BA.
+- Added configuration change logs for hostgroup endpoints.
+- Added missing configuration change logs for host configuration.
+- CLAPI now manages multiline values.
+
+**Authentication**
+
+- Expired tokens are now deleted faster.
+- You can now enable/disable API Tokens.
+
+**Configuration**
+
+- You can now duplicate a locked command.
+- You can now configure several more Broker loggers from the interface.
+
+**Dashboards and widgets**
+
+- Added a "pending refresh" indicator to widgets.
+- Added last update time to widgets and enabled refresh directly from inside widgets.
+- You can now create a public link to a playlist.
+- Added the display of parent hosts statuses when only services are selected in status chart and status grid widgets.
```

Comment:
```suggestion
- Status chart and status grid widgets: when users filter by service / service group / service category and choose to display the data by host, the tiles corresponding to the hosts are displayed correctly instead of displaying "no host found".
```